packages/utils/tool-utils/src/odspTokenManager.ts

```diff
@@ -46,9 +46,7 @@ export type OdspTokenConfig = {
     redirectUriCallback?: (tokens: IOdspTokens) => Promise<string>;
 };
 
-export interface IPushCacheKey { isPush: true }
-export interface IOdspCacheKey { isPush: false; server: string }
-export type OdspTokenManagerCacheKey = IPushCacheKey | IOdspCacheKey;
+export interface OdspTokenManagerCacheKey { isPush: boolean; server: string; }
```

Comment:
nit: since this is now an interface, we might want to name it starting with I
```suggestion
export interface IOdspTokenManagerCacheKey { isPush: boolean; server: string; }
```

Reply:
I fixed it as part of another refactoring that I published in 3rd iteration of this PR